### ISSUE TITLE
User report total time not correct

### ISSUE THREAD

**Issue by schleyk** [jellyfin/jellyfin-plugin-playbackreporting#91]
Hello everyone,
I have noticed a strange behavior in the user statistics If select "weeks" 11 all fine:
![grafik](https://github.com/user-attachments/assets/b01ea15a-4f48-4215-b233-7c2262c11239)

but change to week 12, the "Total Time" resetting to 5 min? But "Total Plays" is correct (see the first user):
![grafik](https://github.com/user-attachments/assets/47a1cecd-9f31-4a8d-b1e3-b93dc6f90de2)

My Settings:
![grafik](https://github.com/user-attachments/assets/fd0d3940-1a77-4805-bc0f-d81e648c20fc)

![grafik](https://github.com/user-attachments/assets/fcb01dda-f383-4aec-9dac-2816d003cd2d)
![grafik](https://github.com/user-attachments/assets/68b83a05-d220-4240-ad89-a09d02cce4f0)

thanks for helping :)

**Comment by Nathan5471** [comment]
I am running into the same issue

**Comment by Sandro905** [comment]
I have the same issue, seeing your situation and mine, I believe it "resets" when it reaches 7 days (1 week), I haven't looked into the code but maybe it's supposed to return "1 week, 5 minutes".
Were you able to fix this?

**Comment by Nathan5471** [comment]
Yeah, I think you might be right. @schleyk if you have more time, is it fixed? I think I might have a slightly different issue then. My time for all time on my account just says "<1 minute," but I have a lot more time than that. It does something similar in the breakdown of user times, but instead I have "00:44:03," and the entire graph is filled.

![Image](https://github.com/user-attachments/assets/8495fd2a-3fa6-49be-a5d3-9edaa2133df2)

![Image](https://github.com/user-attachments/assets/51fed7d0-78e0-49d9-bcd6-03b16323a538)

**Comment by Sandro905** [comment]
@Nathan5471 maybe this is your issue?   https://github.com/jellyfin/jellyfin-plugin-playbackreporting/issues/97
